### DISCUSSION TITLE
Cuál es la diferencia en colocar $VALUE#meta|taskGroup y $VALUE#taskGroup en una SM?

### DISCUSSION THREAD

**Discussion by NicolasPLG** [Cotalker/documentation#48] [2. Questions and Answers] [answered]
Hola revisando con Julio una implementación nos surgió la duda: cuándo colocar en grupo $VALUE#meta|taskGroup y $VALUE#taskGroup

Puede ser que $VALUE#taskGroup sea para los surveys triggers y $VALUE#meta|taskGroup para los requiredSurvey ?

Gracias

**Comment by acedward** [comment, marked as answer]
@NicolasPLG 
Aquí encontrarás la documentación de los contextos

https://doc.cotalker.com/docs/documentation/automation/admin_cotlang#context-of-the-data

Cada lugar donde se invoca un bot parametrizado, se le envíá información al bot, el cual llamamos `contexto`
Este contexto tiene un formato diferente por tipo de llamada y siempre es un JSON

Algunos envian la data:
```
{
   "meta" : { 
      "taskGroup": "1234567890abcdef"
   }
}
```
Aquí debes usar ` $VALUE#meta|taskGroup`

mientras que otros envian:
```
{
  "taskGroup": "1234567890abcdef"
}
```
y aquí debes usar ` $VALUE#taskGroup` 

La única forma de saber cual debes usar, debes mirar la documentación de contextos!

**Reply by acedward** [reply]
@NicolasPLG Vamos a mejorar ahora mismo las explicaciones en esa sección, te avisaré una vez esté actualizado.

**Reply by NicolasPLG** [reply]
Ok @acedward muchas gracias por la información
